src/js/components/Nav/Nav.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { text } from "@storybook/addon-knobs/react";
+import Nav from "./Nav";
+const stories = storiesOf("Nav", module);
+
+stories.add("Default", () => (
+  <div
+    className="u-flex"
+    style={{ backgroundColor: "#FFEFA5", minHeight: "800px" }}
+  >
```

Comment:
Been playing around with CSS Grid in some personal projects, and it is incredibly cool and powerful, and finally has enough browser support to consider adopting. Flex is still awesome for many things, but this layout may lend itself really well to a Grid flow (the Nav and Main sections). Something worth considering for future iterations.